src/contexts/ThemeContext/ThemeContext.jsx

```diff
@@ -1,38 +1,32 @@
 import { useContext, createContext, useState, useEffect } from "react";
 
-const defaultTheme = { theme: "light" };
+const defaultTheme = { theme: "light-theme" };
 
 const ThemeContext = createContext(defaultTheme);
 
 const ThemeProvider = ({ children }) => {
-  const [theme, setTheme] = useState("light");
-  const Toggle = () => {
-    if (theme === "dark") {
-      setTheme("light");
-      document.body.classList.remove("dark-theme");
-      document.body.classList.add("light-theme");
-      localStorage.setItem("theme", "light");
+  const [theme, setTheme] = useState("light-theme");
+  const toggle = () => {
+    if (theme === "dark-theme") {
+      setTheme("light-theme");
+      localStorage.setItem("theme", "light-theme");
     } else {
-      setTheme("dark");
-      document.body.classList.remove("light-theme");
-      document.body.classList.add("dark-theme");
-      localStorage.setItem("theme", "dark");
+      setTheme("dark-theme");
+      localStorage.setItem("theme", "dark-theme");
```

Comment:
even i'm doing same like you, repeating code agian and agian so in next refactoring try to create one custom function to set data in local storage and use that function wherever you want

Reply:
sure...I will check this